traffic_ops/install/bin/postinstall.py

```diff
@@ -0,0 +1,1247 @@
+#!/usr/bin/python3
+"""
+This script is meant as a drop-in replacement for the old _postinstall Perl script.
+
+It does, however, offer several more command-line flags not present in the original, to aid in
+testing.
+
+-a, --automatic               If there are questions in the config file which do not have answers,
+                              the script will look to the defaults for the answer. If the answer is
+                              not in the defaults the script will exit.
+--cfile [FILE]                An input config file used to ask and answer questions.
+--debug                       Enables verbose logging output.
+--defaults [FILE]             Writes out a configuration file with defaults which can be used as
```

Comment:
I actually think POSIX is fine with that, it's GNU non-compliant. At any rate I'm not sure `argparse` can do that, because it parses strings of characters afters a `-` as separate short-form options as per GNU spec. I'll need to look into that.

Reply:
[Looks like argparse does, in fact, support this](https://stackoverflow.com/questions/31127366/single-dash-for-argparse-long-options)